include/wil/win32_helpers.h

```diff
@@ -18,6 +18,22 @@
 #include <PathCch.h>
 #include <objbase.h>
 
+// detect std::bit_cast
+#ifdef __has_include
+#  if __has_include(<version>)

```

Comment:
Mostly nitpicking at this point, but this could be simplified to just `#if __has_include(<bit>)` since that header (also) defines the feature test macro

Reply:
I was thinking in case there is standard library with a `<bit>` which doesn't have `std::bit_cast` but supports other functions from that header, we would avoid bringing it in entirely only to not use it.